### PR TITLE
Data migration to populate `FlowSession.wait_started_on` and `wait_expires_on`

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: temba/flows/migrations/0264_populate_session_wait_expires.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+                    # if run didn't have a expires_on.. use default for flow type
+                    if not session.wait_expires_on:
+                        default_expires_after = EXPIRES_DEFAULTS[session.session_type]
+                        session.wait_expires_on = session.wait_started_on + timedelta(minutes=default_expires_after)
+
+                    session.save(update_fields=("wait_started_on", "wait_expires_on"))
+                    num_updated += 1
+                else:
+                    # if we don't have a waiting run.. we're in an invalid state so expire this session
+                    session.status = "X"
```
Comment (rowanseymour) [inline temba/flows/migrations/0264_populate_session_wait_expires.py:60]: our prod boxes have a hundred or so sessions in this state, all at least a year old

File: temba/flows/migrations/0264_populate_session_wait_expires.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    num_updated = 0
+    num_expired = 0
+
+    while True:
+        with transaction.atomic():
+            # get a batch of waiting sessions which don't have wait_started_on or wait_expires_on set
+            batch = list(
+                FlowSession.objects.filter(status="W")
+                .filter(Q(wait_started_on__isnull=True) | Q(wait_expires_on__isnull=True))
+                .only("id", "wait_started_on", "wait_expires_on")[:1000]
```
Comment (norkans7) [inline temba/flows/migrations/0264_populate_session_wait_expires.py:30]: we need to include `session_type` as we use that on  https://github.com/nyaruka/rapidpro/pull/3641/files#diff-6655ebb6d3df0e2d795364fa9009f8116330e5de4fa8f94314c583adcbd6a9a0R53

Comment (rowanseymour) [reply temba/flows/migrations/0264_populate_session_wait_expires.py:30]: You have eagle eyes!

**Review by norkans7 — APPROVED** [review]

File: temba/flows/migrations/0264_populate_session_wait_expires.py (hunk 2 above)
Comment (norkans7) [reply temba/flows/migrations/0264_populate_session_wait_expires.py]: Hahahahah :)

**Pull request merged**